metadata/src/test/java/org/apache/kafka/image/ImageDowngradeTest.java

```diff
@@ -103,7 +103,7 @@ public void testPremodernVersion() {
     @Test
     public void testPreControlledShutdownStateVersion() {
         writeWithExpectedLosses(MetadataVersion.IBP_3_3_IV2,
-                Arrays.asList(
+                Collections.singletonList(
```

Comment:
Can we use the same alignment as in the other methods?

Reply:
Updating the same. It will cause some additional indentation changes as I see the file has some weird indentation right now

Reply:
I meant shifting this whole block to the left so it's aligned like the `testPremodernVersion()` and `testPreZkMigrationSupportVersion()` methods for example

Reply:
Makes sense.
I had updated this method and the `testDirectoryAssignmentState()` method too as it also had a similar indentation. Hope it makes sense now.